app/routes/subscribe.js

```diff
@@ -0,0 +1,6 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class SubscribeRoute extends Route {
+  @service login;
```

Comment:
If we are not using login service here, why its `injected`. 

you can remove it

Reply:
removed